### Множество. Последовательность.

##### Как относятся друг к другу множество и последовательность? (в ответе использовать слова типа: часть, целое, общее, частное, родитель, дочерний субъект и т.д.)

Последовательность это частный случай множеств. В последовательности могут быть только вещественные числа, последовательность упорядоченна.

### Последовательность.

#### 1. Даны 4 последовательности. Необходимо:

- исследовать их на монотонность;
- исследовать на ограниченность;
- найти пятый по счету член.


### а) $\{a_n\}_{n=1}^\infty=2^n - n$

In [1]:
start_n = 1
n = 5
a = [0]*n
for i in range(start_n, start_n + n):
    a[i-start_n] = 2**i-i
print(a)

[1, 2, 5, 12, 27]


__Ответ__: Монотонно возрастающая, неограниченная, $a_5=27$

### б) $\{b_n\}_{n=2}^\infty=\frac{1}{1 - n}$

In [2]:
start_n = 2
n = 5
a = [0]*n
for i in range(start_n, start_n + n):
    a[i-start_n] = 1/(1-i)
print(a)

[-1.0, -0.5, -0.3333333333333333, -0.25, -0.2]


__Ответ__: Монотонно возрастающая, ограниченная 0, $a_5=-0.2$

### в) $\{c_n\}_{n=1}^\infty=-1^n + \sqrt{2n}$

In [3]:
start_n = 1
n = 5
a = [0]*n
for i in range(start_n, start_n + n):
    a[i-start_n] = (-1)**i + (2*i)**0.5
print(a)

[0.41421356237309515, 3.0, 1.4494897427831779, 3.8284271247461903, 2.1622776601683795]


__Ответ__: Немонотонная, неограниченная, $a_5=-1 + \sqrt10$

### г) $\{d_n\}_{n=1}^\infty=(-1)^{2n} + \frac{1}{n^2}$

In [4]:
start_n = 1
n = 5
a = [0]*n
for i in range(start_n, start_n + n):
    a[i-start_n] = (-1)**(2*i) + 1/(i**2)
print(a)

[2.0, 1.25, 1.1111111111111112, 1.0625, 1.04]


__Ответ__: Монотонно убывающая, ограниченная 1, $a_5=1.04$

#### 2. Найти 12-й член заданной неявно последовательности

### $a_1=128, a_{n+1} - a_n =6$

$a_1=128, a_{n+1} - a_n =6$ можно записать в виде $a_{n+1} = a_n + 6$ или $a_n=6(n-1)+128$

In [5]:
n = 12
a = [0]*n

a[0]=128

for i in range(1, n, 1):
    a[i] = a[i-1]+6
print(a)

for i in range(n):
    a[i] = 6*i+128
print(a)

[128, 134, 140, 146, 152, 158, 164, 170, 176, 182, 188, 194]
[128, 134, 140, 146, 152, 158, 164, 170, 176, 182, 188, 194]


__Ответ__: $a_{12}=194$